app/views/supervisor/dashboard/index.html.slim

```diff
@@ -0,0 +1,10 @@
+h1 = "Hi #{current_user.name}!"
+h2 This is your brandnew Supervisor Dashboard - with love from Team Cheesy
+
+hr
+
+- for t in supervised_teams.each do
+  h4 = link_to t.name, t
+  h4 Last checked at:
+  p = t.last_checked_at
```

Comment:
@F3PiX, the date shows as "2015-09-01". Should we change this? And if yes, where?